docs/Splashkit/Applications/Arcade Machines/Arcade Machine Setup/AddGameOverNetwork.md

```diff
@@ -0,0 +1,116 @@
+# Adding Games To Arcade Machine
+
+## Pre Requisites:
+### WiFi Access Point
+This guide assumes that there is an existing WiFi Access Point on the Arcade Machine you are trying to copy games to.  You should be able to verify this by looking for nearby WiFi networks called “Arcade`X`” where `X` is the number of the Arcade Machine. 
+
+If you are unable to see the WiFi network, or you need to set it up follow this guide: [link](https://github.com/thoth-tech/documentation/blob/main/docs/Splashkit/Applications/Arcade%20Machines/Arcade%20Machine%20Setup/Setup%20Arcade%20Machine.md#setup-wifi-access-point-optional).
+
+### SSH Utilities
+In order transfer the files from your local machine to the Arcade Machine you will need to use either SCP or SFTP. Both applications are part of the SSH suite and come pre-installed on most modern operating systems. In order to test that these are installed open a terminal (If you're running **Windows** make sure you use **PowerShell**!). From the terminal window type ```scp``` and/or ```sftp```. You should output similar to the below prompting usage of these commands. If you do you're good to go! 
```

Comment:
SFTP and SCP have rather different interfaces (as shown below) however the example below will only apply to scp which could cause confusion if you used sftp instead.

Additionally I don't see a need for typing both scp and sftp as you'll just end up with an invalid command. Hence I would suggest rewording this bit to say something like "In the terminal type either `scp` or `sftp`, you should get an output similar to the respective example below"

Reply:
When I was testing this out just typing "sftp" without any arguments won't load the program, it just prints out usage syntax like the SCP example.  

The intention was that users would type "scp", then enter then "sftp" then enter to test both programs, but i see how the sentence "....`scp` and/or `sftp`.' could be interpreted ambiguously.  I've reworded it slightly to make it more clear.